### PR TITLE
Truncate vector in logs

### PR DESCRIPTION
**Pull request by willtai**
# Description

This PR changes what is logged from the retrieved items such that the vector embeddings are not logged and only displayed as "...".

## Type of Change
- [x] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Documentation update
- [ ] Project configuration change

## Complexity
Low

## How Has This Been Tested?
- [ ] Unit tests
- [ ] E2E tests
- [x] Manual tests

# Checklist

The following requirements should have been met (depending on the changes in the branch):

- [ ] Documentation has been updated
- [ ] Unit tests have been updated
- [ ] E2E tests have been updated
- [ ] Examples have been updated
- [ ] New files have copyright header
- [ ] CLA (https://neo4j.com/developer/cla/) has been signed
- [ ] CHANGELOG.md updated if appropriate

### CODE REVIEW

File: src/neo4j_graphrag/retrievers/hybrid.py [hunk 1]
```diff
@@ -348,9 +352,13 @@ def get_search_results(
...
             neo4j_version_is_5_23_or_above=self.neo4j_version_is_5_23_or_above,
         )
-
-        logger.debug("HybridCypherRetriever Cypher parameters: %s", parameters)
-        logger.debug("HybridCypherRetriever Cypher query: %s", search_query)
+        sanitized_parameters = copy.deepcopy(parameters)
+        sanitized_parameters["query_vector"] = (
+            "..." if "query_vector" in sanitized_parameters else None
+        )
+        print("logger", logger)
```
Comment (stellasia) [inline src/neo4j_graphrag/retrievers/hybrid.py:359]: Oups

File: src/neo4j_graphrag/retrievers/hybrid.py [hunk 2]
```diff
@@ -193,8 +194,11 @@ def get_search_results(
             embedding_node_property=self._embedding_node_property,
             neo4j_version_is_5_23_or_above=self.neo4j_version_is_5_23_or_above,
         )
-
-        logger.debug("HybridRetriever Cypher parameters: %s", parameters)
+        sanitized_parameters = copy.deepcopy(parameters)
+        sanitized_parameters["query_vector"] = (
```
Comment (stellasia) [inline src/neo4j_graphrag/retrievers/hybrid.py:198]: Looks good. I'm just wondering if that would be misleading to display the `query_vector` as parameter (even as None) when it's not in the actual parameters? Maybe I'm just overthinking :D

Comment (willtai) [reply src/neo4j_graphrag/retrievers/hybrid.py]: https://github.com/neo4j/neo4j-graphrag-python/pull/192/commits/64dc99d15149f998a94807eed677d1bd043a9ca1

I think you're right, it's misleading

**Review by stellasia — APPROVED** [review]
Thank you!

**Pull request merged**